apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Trace/CloudTraceExtension.cs

```diff
@@ -121,7 +124,15 @@ public static void UseGoogleTrace(this IApplicationBuilder app)
         /// </summary>
```

Comment:
I think you're right though.  Otherwise it's putting too much on the user to figure out scoping of things.